work-in-progress/WIP-json-schema-release-notes.md

```diff
@@ -0,0 +1,52 @@
+---
+title: JSON Schema Draft-07 Release Notes
+layout: page
+---
+
+There are a few breaking changes with newer keywords based on community feedback,
```

Comment:
I want to be careful with the phrase "breaking changes."  I think this just refers to `dependencies` and `definitions`?

We deliberately left a compatibility strategy in place for both of these by reserving the old keywords in the meta-schema.  So `definitions` isn't broken at all.  It's no longer recommended, but it SHOULD still work.  `dependencies` is likely to break except when implementations specifically offer a compatibility mode.  So I'd be OK with saying the `dependencies` change is breaking, but maybe note that we did it in a way that allows for a compatibility mode?  I dunno, don't want to get too complicated here.

Reply:
Done